#ifndef STATEMACHINENODE
#define STATEMACHINENODE

#include <ObstacleGridMap.hpp>
#include <Utils.hpp>
#include <localization_msgs/MatchingResult.h>
#include <planner_msgs/Pose2DArray.h>
#include <planner_msgs/PlanningRequest.h>
#include <visualization_msgs/MarkerArray.h>
#include <jsk_rviz_plugins/OverlayText.h>
#include <std_msgs/Float32.h>

#endif
